src/test-integration/java/edition/academy/seventh/serivce/DataProviderForJsonParser.java

```diff
@@ -0,0 +1,80 @@
+package edition.academy.seventh.serivce;
+
+import org.testng.annotations.DataProvider;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataProviderForJsonParser {
+
+  @DataProvider
+  public static Object[][] dataProviderForJsonResponse() {
+    return new Object[][] {
+      {
```

Comment:
Zwracana lista może się zmienić nawet jutro, czyli nawet jutro ten test może nie przejść? :o

Reply:
Jeżeli to jest pod test intergracyjny sprawdzałbym intergracje z serwisem na podstawie zwracanego kodu, a niekoniecznie zwracanej zawartości w tym wypadku :)